cmake 3.20: "file does not recognize sub-command COPY_FILE"

We have agreed to support cmake 3.20 for our cmake builds. I'm getting ready to create a CI job to build with cmake 3.20 and I tested a build with this in rockylinux:8. I observed this cmake failure while running `--install`:

https://ci.trafficserver.apache.org/job/Github_Builds/job/rocky/2979/console

```
+ cmake --install cmake-build-quiche
-- Install configuration: "Debug"
-- Installing: /tmp/ats_quiche/lib64/libfastlz.so
...
-- Set runtime path of "/tmp/ats_quiche/libexec/trafficserver/healthchecks.so" to "/tmp/ats_quiche/lib"
-- Installing config: /tmp/ats_quiche/etc/trafficserver/records.yaml
CMake Error at cmake/install_configs.cmake:33 (file):
  file does not recognize sub-command COPY_FILE
Call Stack (most recent call first):
  cmake-build-quiche/configs/cmake_install.cmake:54 (include)
  cmake-build-quiche/cmake_install.cmake:132 (include)
```

In the meantime, I'll revert PR CI Rocky jobs to use automake.

## Reproduction steps:

```sh
docker pull ci.trafficserver.apache.org/ats/rockylinux:8
docker run --rm -it ci.trafficserver.apache.org/ats/rockylinux:8 /bin/bash

# Now, from inside docker.
source /opt/rh/gcc-toolset-11/enable

cd /var/tmp
git clone https://github.com/apache/trafficserver.git
cd trafficserver

cmake -B cmake-build-quiche -DCMAKE_COMPILE_WARNING_AS_ERROR=ON -DENABLE_QUICHE=ON -DCMAKE_BUILD_TYPE=Debug -DBUILD_EXPERIMENTAL_PLUGINS=ON -Dquiche_ROOT=/opt/quiche -DOPENSSL_ROOT_DIR=/opt/boringssl -DCMAKE_INSTALL_PREFIX=/tmp/ats_quiche
cmake --build cmake-build-quiche -j4 -v
cmake --install cmake-build-quiche

```

--- a/cmake/install_configs.cmake
+++ b/cmake/install_configs.cmake
@@ -30,5 +30,11 @@ foreach(CONFIG_FILE ${CONFIG_FILES})
     set(DEST_FILE "${DEST_FILE}.default")
   endif()
   message(STATUS "Installing config: ${DEST_FILE}")
-  file(COPY_FILE ${CONFIG_FILE} ${DEST_FILE})
+  # Prefer copy_file but we need to support 3.20 which doesn't have this feature so fall back to
+  # configure_file
+  if(CMAKE_MINOR_VERSION GREATER 20)
+    file(COPY_FILE ${CONFIG_FILE} ${DEST_FILE})
+  else()
+    configure_file(${CONFIG_FILE} ${DEST_FILE} COPYONLY)
+  endif()
 endforeach()

